packages/rich-text/src/plugins/shared/EmbeddedInlineUtil.ts

```diff
@@ -32,29 +32,28 @@ export function getWithEmbeddedEntryInlineEvents(
   };
 }
 
-const getLink = (nodeType: INLINES, entity) => {
-  if (nodeType === INLINES.EMBEDDED_RESOURCE) {
-    return {
-      urn: entity.sys.urn,
-      type: 'ResourceLink',
-      linkType: 'Contentful:Entry',
-    };
-  }
+const createInlineEntryNode = (nodeType, entity) => {
```

Comment:
I combined two helpers because they were so simple. But I realized I did it differently for the embedded node and the block node so I just aligned both and kept the helper.